python/ambassador/fetch/ingress.py

```diff
@@ -104,6 +104,20 @@ def _update_status(self, obj: KubernetesObject) -> None:
                 f"Not reconciling Ingress {obj.name}: observed and current statuses are in sync"
             )
 
+    def _resolve_service_port_number(self, namespace, service_name, service_port):
+        for k8s_svc in self.service_dep.discovered_services.values():
```

Comment:
discovered_services is a dict, can't we just use `self.service_dep.discovered_services.get(f"{service_name}.{namespace}", None)`? 

If the cluster has a larger number of services we potentially would have to do a lot of looping here.

Reply:
My bad 🤦  will update soon